### PR TITLE
docs: db migration

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add simple description on how to run database migration

## Testing

Local previewed

### CODE REVIEW

File: docs/docs/recipes/deployment/README.md [hunk 1]
```diff
@@ -90,4 +90,6 @@ You are all set. Open the browser and visit your domain URL, you should be able
 
 If we don't mention it in the changelog, you can feel free to upgrade Logto without changing your code or database schema. Our API follows [semver](https://semver.org/).
 
-We'll build a smooth database migration process and demonstrate it in our changelog if a schema change is inevitable. You'll be able to do that with chill.
+### Database migration
+
+If a schema change is inevitable, we will provide a migration script. You don't need to care about it, simply run "pnpm migration-deploy" to upgrade your database schema.
```
Comment (charIeszhao) [inline docs/docs/recipes/deployment/README.md:95]:
```suggestion
If a schema change is inevitable, we will provide a migration script. Simply run "pnpm migration-deploy" with ease to upgrade your database schema.
```

File: docs/docs/recipes/deployment/README.md [hunk 2]
```diff
@@ -90,4 +90,6 @@ You are all set. Open the browser and visit your domain URL, you should be able
 
 If we don't mention it in the changelog, you can feel free to upgrade Logto without changing your code or database schema. Our API follows [semver](https://semver.org/).
 
-We'll build a smooth database migration process and demonstrate it in our changelog if a schema change is inevitable. You'll be able to do that with chill.
+### Database migration
```
Comment (gao-sun) [inline docs/docs/recipes/deployment/README.md:93]: should be "alternation"? then we can separate from "migrate from other projects"

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/deployment/README.md [hunk 3]
```diff
@@ -90,4 +90,6 @@ nginx -s reload
 
 除非我们在 changelog 里特意提出，你都无需变更代码和数据库即可轻松升级 Logto。我们的 API 遵循 [semver](https://semver.org/) 标准。
 
-如果数据库变更无法避免，我们将提供一个顺滑的数据库迁移过程，并在 changelog 中详细描述。你将能轻松完成它。
+### 数据库迁移
+
+如果数据库变更无法避免，我们将提供迁移脚本。你无需关心，只需运行 `pnpm migration-deploy` 即可升级数据库。
```
Comment (charIeszhao) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/deployment/README.md:95]:
```suggestion
如果数据库变更无法避免，我们将提供迁移脚本。只需简单运行 `pnpm migration-deploy` 即可轻松升级数据库。
```

**Review by charIeszhao — APPROVED** [review]

**Review by darcyYe — APPROVED** [review]

File: docs/docs/recipes/deployment/README.md (hunk 2 above)
Comment (wangsijie) [reply docs/docs/recipes/deployment/README.md:93]: Sure, then I'll also change `migration` from code later.

**Review by gao-sun — APPROVED** [review]

**Pull request merged**